cmd/mdatagen/internal/sampleprocessor/metadata.yaml

```
@@ -66,3 +66,24 @@ resource_attributes:
     enabled: true
     warnings:
       if_enabled: This resource_attribute is deprecated and will be removed soon.
+
+feature_gates:
+  useOTTLBridge:
+    id: useOTTLBridgeGate
+    stage: StageAlpha
+    description: When enabled, filterlog will convert filterlog configuration to OTTL and use filterottl evaluation
+    from_version": 1.0.0
```

There is an error in CI, I think because of this:
```suggestion
    from_version: 1.0.0
```

Even then, I think it should look like this to pass your regex

```suggestion
    from_version: v1.0.0
```

and same below for `to_version`